components/AddNew/AddNewForm.js

```diff
@@ -0,0 +1,106 @@
+import { nanoid } from "nanoid";
+import styled from "styled-components";
+// import format from "date-fns/format";
+
+//Symbol
+import { BsPlusCircleFill } from "react-icons/bs";
+
+// Funktion
+
+export default function AddNewForm({ onAddAppointment }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const form = event.target;
+    const text = form.text.value;
+    const name = form.name.value;
+    const date = form.date.value;
+    // const date = format(Date(form.date.value), "dd'.'MM'.'yyyy");
+    const clock = form.clock.value;
+
+    const newAppointment = {
+      id: nanoid(),
+      text: text,
+      name: name,
+      date: date,
+      clock: clock,
+    };
+
+    onAddAppointment(newAppointment);
+
+    form.reset();
+  }
+
+  return (
+    <>
+      <StyledEditForm onSubmit={handleSubmit}>
+        <InputWrapper>
+          <label htmlFor="text">
+            <ScreenReaderOnly>TEXT</ScreenReaderOnly>
```

Comment:
Ist das ein Ding, dass man sowas macht? Ich hätte stattdessen aria-labels verwendet, dann musst du nichts weg-stylen. Wenn du es so lassen willst, dann würde ich es aber glaube ich auch nicht in Großbuchstaben schreiben, oder ist das so Konvention?